addon/components/discover-page/component.js

```diff
@@ -348,11 +354,6 @@ export default Ember.Component.extend(Analytics, hostAppName, {
             this.setActiveFiltersAndReload('activeFilters.providers', filter.split('OR'));
         }
     })),
-    processedTypes: Ember.computed('types', function() {
```

Comment:
Hey! Looked over this for a while, and can't seem to figure out what they did in the first place. Could you provide some insight into why they were here and why they are no longer needed?